src/client/app/index.tsx

```diff
@@ -14,9 +14,23 @@ import './styles/index.css';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import initScript from './initScript';
 
-// Creates and applies thunk middleware to the Redux store, which is defined from the Redux reducers
+// Creates and applies thunk middleware to the Redux store, which is defined from the Redux reducers.
+// For now we are enabling Redux debug tools on production builds. If had a good way to only do this
+// when not in production mode then maybe we should remove this but it does allow for debugging.
+// Comment this out if enabling traces below.
 const store = createStore(reducers, composeWithDevTools(applyMiddleware(thunkMiddleware)));
```

Comment:
What do you think about replacing this line with:
`const store = createStore(reducers, applyMiddleware(thunkMiddleware));`?
It will turn off the Redux debug tools, which is not necessary for production mode. Developers can comment this line out and uncomment the two lines below to enable the traces and debug tools.

Reply:
I thought about doing that. There was some discussion on the web about the advantages of leaving debug tools on in production environments without impacting performance if the developer tools are not being used. It also allows site users to try to trace down issues. I thought I would leave it for now so the project and users can more readily debug a live site. I'm open to turning it off if it impacts sites or there are security concerns (which could be addressed by hiding that state).

Reply:
Thank you for your explanation. I agree with you. This PR is good to merge!